include/swift/AST/DiagnosticsParse.def

```diff
@@ -1109,7 +1109,7 @@ NOTE(suggest_removing_else,none,
 ERROR(expected_condition_guard,PointsToFirstBadToken,
       "expected expression, var, let or case in 'guard' condition", ())
 ERROR(missing_condition_after_guard,none,
-      "missing condition in an 'guard' statement", ())
+      "missing condition in a 'guard' statement", ())
```

Comment:
IMO this would read better as:

```suggestion
      "missing condition in 'guard' statement", ())
```

what do you think? We could change `missing_condition_after_if` to match as well.

Reply:
Sounds good to me